modules/home/software/nodejs.nix

```diff
@@ -1,5 +0,0 @@
-{ config, ... }:
-
-{
-  home.file.".npmrc".text = "prefix=${config.home.homeDirectory}/.filesystem";
```

Comment:
I would leave this (and the it's counterpart in zsh). Without it, if you ever do `npm install -g <some package>` you'll get an error because it will try to install in the nix store.

Reply:
Yes, i was planning on putting it back in